cmd/charmd/config.yaml

```diff
@@ -20,3 +20,5 @@ identity-api-url: http://localhost:8081
 #stats-cache-max-age: 1h
 #request-timeout: 500ms
 #search-cache-max-age: 0s
+# Uncomment to test with a terms service running locally
```

Comment:
Move this up after identity, as this is external service specification. In the end we have charm store attributes only.